tests/experiment/termsearch.py

```diff
@@ -38,20 +38,15 @@ def main():
     #      for items in term['_source']['existing_ids']:
     #          if items['preferred']=='1':
     #              preferred_url=items['iri']
-    #      print("Label = %s \t Definition = %s \t Preferred URL = %s " %(term['_source']['label'],term['_source']['definition'],preferred_url))
+    #      print(f"Label = {term['_source']['label']} \t Definition = {term['_source']['definition']} \t Preferred URL = {preferred_url} ")
 
     # example of uber elastic search query returns dictionary of label, definition, and preferred_url
     print("\n\n-------------------------------------------")
     print("Example uber elastic search:")
     results = Utils.GetNIDMTermsFromSciCrunch(args.key, args.query_string)
     for key, _ in results.items():
         print(
-            "Label: %s \t Definition: %s \t Preferred URL: %s "
-            % (
-                results[key]["label"],
-                results[key]["definition"],
-                results[key]["preferred_url"],
-            )
+            f"Label: {results[key]['label']} \t Definition: {results[key]['definition']} \t Preferred URL: {results[key]['preferred_url']} "
```

Comment:
May be improve it while at it as: 

```suggestion
            "Label: {label} \t Definition: {definition} \t Preferred URL: {preferred_url} ".format(**results[key])
```
to make shorter and thus easier to grasp visually, and make it explicit that we are reporting for the same `key`

Reply:
Done: e3616fb